### PR TITLE
Fix bookmarking aspect goes invisible

### PR DESCRIPTION
**Pull request by miozune**

### CODE REVIEW

File: src/main/java/ru/timeconqueror/tcneiadditions/nei/TCAspectStringifyHandler.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import com.djgiannuzz.thaumcraftneiplugin.items.ItemAspect;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraftforge.fluids.FluidStack;
+import thaumcraft.api.aspects.Aspect;
+import thaumcraft.api.aspects.AspectList;
+
+public class TCAspectStringifyHandler implements IStackStringifyHandler {
+    @Override
+    public NBTTagCompound convertItemStackToNBT(ItemStack stack, boolean saveStackSize) {
```
Comment (slprime) [inline src/main/java/ru/timeconqueror/tcneiadditions/nei/TCAspectStringifyHandler.java:14]: please save Count to nbt if saveStackSize is true. see DefaultStackStringifyHandler

File: src/main/java/ru/timeconqueror/tcneiadditions/nei/TCAspectStringifyHandler.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            return null;
+        }
+        Aspect aspect = aspectList.getAspects()[0];
+        NBTTagCompound nbtTag = new NBTTagCompound();
+        nbtTag.setString("TCAspect", aspect.getTag());
+        return nbtTag;
+    }
+
+    @Override
+    public ItemStack convertNBTToItemStack(NBTTagCompound nbtTag) {
```
Comment (slprime) [inline src/main/java/ru/timeconqueror/tcneiadditions/nei/TCAspectStringifyHandler.java:30]: and here need restore stackSize

File: src/main/java/ru/timeconqueror/tcneiadditions/nei/TCAspectStringifyHandler.java (hunk 1 above)
Comment (slprime) [reply src/main/java/ru/timeconqueror/tcneiadditions/nei/TCAspectStringifyHandler.java:14]: ezgif com-gif-maker (8)
Count needed to keep this behavior

In the future need save correct aspect amount

Comment (miozune) [reply src/main/java/ru/timeconqueror/tcneiadditions/nei/TCAspectStringifyHandler.java:14]: Ah, I got it

**Review by slprime — APPROVED** [review]

**Pull request merged**